src/graph.py

```diff
@@ -59,36 +60,26 @@ def build_graph(self, loader):
             "check_for_remaining_leads",
             nodes.check_if_there_more_leads,
             {
-                "Found leads": "fetch_linkedin_profile_data",  # Proceed if leads are found
+                "Found leads": "generate_prompt_from_pitch_deck",  # Proceed if leads are found
                 "No more leads": END  # Terminate if no leads remain
             }
         )
 
         # Research phase transitions
-        graph.add_edge("fetch_linkedin_profile_data", "review_company_website")
+        graph.add_edge("generate_prompt_from_pitch_deck", "fetch_linkedin_profile_data")
+        graph.add_edge("fetch_linkedin_profile_data", "analyze_lead_social_profile")
+        graph.add_edge("analyze_lead_social_profile", "review_company_website")
         graph.add_edge("review_company_website", "collect_company_information")
 
         # Collect company information and branch into various analyses
-        graph.add_edge("collect_company_information", "analyze_blog_content")
         graph.add_edge("collect_company_information", "analyze_social_media_content")
         graph.add_edge("collect_company_information", "analyze_recent_news")
 
         # Analysis results converge into generating reports
-        graph.add_edge("analyze_blog_content", "generate_digital_presence_report")
-        graph.add_edge("analyze_social_media_content", "generate_digital_presence_report")
-        graph.add_edge("analyze_recent_news", "generate_digital_presence_report")
-        graph.add_edge("generate_digital_presence_report", "generate_full_lead_research_report")
-
-        # Scoring phase with conditional qualification check
-        graph.add_edge("generate_full_lead_research_report", "score_lead")
-        graph.add_conditional_edges(
-            "score_lead",
-            nodes.check_if_qualified,
-            {
-                "qualified": "generate_custom_outreach_report",  # Proceed if lead is qualified
-                "not qualified": "save_reports_to_google_docs"  # Save reports and exit if lead is unqualified 
-            }
-        )
+        graph.add_edge("analyze_social_media_content", "generate_company_profile_report")
+        graph.add_edge("analyze_recent_news", "generate_company_profile_report")
+        graph.add_edge("generate_company_profile_report", "generate_full_lead_research_report")
```

Comment:
I think that we can probably stream line this and go directly to custom outreach reports. I would do `generate_company_profile_report` -> `generate_custom_outreach_report`. Also, perhaps I missed this, but the `generate_custom_outreach_report` does not use the VC profile that was created in earlier nodes. This should be there.